auto3dseg/algorithm_templates/dints/configs/hyper_parameters_search.yaml

```diff
@@ -13,22 +13,22 @@ searching:
   num_epochs: 1000
   num_epochs_per_validation: 20
   num_warmup_epochs: 500
-  num_patches_per_image: 1
+  num_crops_per_image: 1
   num_sw_batch_size: 2
   num_workers: 6
   num_cache_workers: 8
   output_classes: null
   overlap_ratio: 0.625
-  patch_size: null
-  patch_size_valid: null
+  roi_size: null
+  roi_size_valid: null
   ram_cost_factor: 0.8
-  resample_to_spacing: null
+  resample_resolution: null
   softmax: true
   cache_rate: 1
   train_cache_rate: "$@searching#cache_rate"
   validate_cache_rate: "$@searching#cache_rate"
   transforms:
-    resample_to_spacing: "$@searching#resample_to_spacing"
+    resample_to_spacing: "$@searching#resample_resolution"
```

Comment:
The key "resample_to_spacing" should also be changed to "resample_resolution"?

Reply:
Thanks! I consolidated the naming for this one to avoid confusion.